src/services/codefixes/helpers.ts

```diff
@@ -608,6 +619,43 @@ export function typeToAutoImportableTypeNode(checker: TypeChecker, importAdder:
     return getSynthesizedDeepClone(typeNode);
 }
 
+function endOfRequiredTypeParameters(checker: TypeChecker, type: GenericType, fullTypeArguments: readonly Type[]) : number {
+    if (fullTypeArguments !== type.typeArguments!) {
+        throw new Error('fullTypeArguments should be set')

```

Comment:
Instead of throwing, use `Debug.fail`. Or maybe `Debug.assertNotEqual` (if that variant exists).

Reply:
I simplified the signature here so it can just use Debug.assert